src/dos/program_imgmount.cpp

```diff
@@ -36,12 +36,62 @@
 #include "string_utils.h"
 #include "../ints/int10.h"
 
+void IMGMOUNT::ListImgMounts(void)
+{
+	const std::string header_drive = MSG_Get("PROGRAM_MOUNT_STATUS_DRIVE");
+	const std::string header_name = MSG_Get("PROGRAM_MOUNT_STATUS_NAME");
+	const std::string header_label = MSG_Get("PROGRAM_MOUNT_STATUS_LABEL");
+	const std::string header_slot = MSG_Get("PROGRAM_MOUNT_STATUS_SLOT");
+
+	const int term_width = real_readw(BIOSMEM_SEG, BIOSMEM_NB_COLS);
+	const auto width_1 = static_cast<int>(header_drive.size());
+	const auto width_3 = std::max(minimum_length,
+	                              static_cast<int>(header_label.size()));
+	const auto width_4 = std::max(minimum_length,
+	                              static_cast<int>(header_slot.size()));
```

Comment:
Suggest using string's `.length()` instead of `.size()`.
Length also makes it read more consistent when comparing against `minimum_length`.

Reply:
Done.